src/components/FavoritesBtn.tsx

```diff
@@ -9,14 +9,14 @@ type FavoritesBtnProps = {
 };
 
 const FavoritesBtn: React.FC<FavoritesBtnProps> = ({ movieId }) => {
-  const [isFavorite, setIsFavorite] = useState(api.isIdInFavorites(movieId));
- 
+  const [isFavorite, setIsFavorite] = useState(!api.isIdInFavorites(movieId));
```

Comment:
Logical error - isFavorite should be clearly set to result isIdInFavorites().

Reply:
![image](https://user-images.githubusercontent.com/41084902/115024161-616b4580-9ec8-11eb-8998-b15ed1b24836.png)

Reply:
in this method was "!" namely due to this reason

Reply:
So this should be fixed

Reply:
Still valid - if Id in favorites list it should evaluate isFavorites to true...